clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -148,6 +148,16 @@ New checks
   Flags coroutines that suspend while a lock guard is in scope at the
   suspension point.
 
+- New :doc:`google-cpp-init-class-members
+  <clang-tidy/checks/google/cpp-init-class-members>` check.
+
+  Checks that class members are initialized in constructors (implicitly or
```

Comment:
One sentence is enough.

Reply:
fixed in a0d227088b9d099b23ed68af80714e2f96a7a2a8. Thanks!